straxen/plugins/daqreader.py

```diff
@@ -141,12 +143,32 @@ def _chunk_paths(self, chunk_i):
                 result.append(False)
         return tuple(result)
 
+    @staticmethod
+    def _partial_chunk_to_thread_name(partial_chunk):
+        """Convert name of part of the chunk to the thread_name that wrote it"""
+        return '_'.join(partial_chunk.split('_')[-1])
```

Comment:
`reader0_reader_0_threadid` -> `t_h_r_e_a_d_i_d`, is that what we are interested in, or should it be `[:-1]`?